ovs/extensions/db/arakoon/ArakoonInstaller.py

```diff
@@ -83,6 +83,7 @@ def __init__(self, cluster_id, plugins=None):
         self.cluster_id = cluster_id
         self._dir = ArakoonClusterConfig.ARAKOON_CONFIG_DIR.format(self.cluster_id)
         self.filename = ArakoonClusterConfig.ARAKOON_CONFIG_FILE.format(self.cluster_id)
+        self._extra_globals = {'__tainted_tlog_entries_per_file': 5000}
```

Comment:
Yes, under no circumstances do you want to change this setting for an already existing cluster.
Also all nodes in the cluster should be configured in the same way regarding this property.
(Which I think from looking at your code should be the case)